Tools/python DM botany to json/Transport.py

```diff
@@ -1,68 +1,137 @@
+    genes = genes.replace('/datum/plant_gene/trait/battery','"Capacitive Cell Production"')
+    genes = genes.replace('/datum/plant_gene/trait/plant_type/weed_hardy','"Weed Adaptation"')
+    genes = genes.replace('/datum/plant_gene/trait/stinging','"Hypodermic Prickles"')
+    genes = genes.replace('/datum/plant_gene/trait/glow/shadow','"Shadow Emission"')
+    genes = genes.replace('/datum/plant_gene/trait/glow/red','"Red Electrical Glow"')
+    genes = genes.replace('/datum/plant_gene/trait/cell_charge','"Electrical Activity"')#
+    genes = genes.replace('/datum/plant_gene/trait/glow/berry','"Strong Bioluminescence"')
+    genes = genes.replace('/datum/plant_gene/trait/glow','"Bioluminescence"')
+    genes = genes.replace('/datum/plant_gene/trait/noreact','"Separated Chemicals"')
+    genes = genes.replace('] | [',',')
+    genes = genes.replace('list(','[')
+    return genes.replace(')',']')
+
+def mutates_replace(line):
+    mutates_into = line
+    mutates_into = mutates_into.replace('mutatelist','mutates_into')
+    mutates_into = mutates_into.replace('/obj/item/seeds/','')
+    mutates_into = mutates_into.replace('/','_')
+    mutates_into = mutates_into.replace(', ',',')
+    mutates_into = mutates_into.replace(',','","')
+    mutates_into = mutates_into.replace('list(','["')
+    return mutates_into.replace(')','"]')
+
+
+def chemical_replace(line):
+    chemical_content = line
+    chemical_content = chemical_content.replace('= list','@list')
+    chemical_content = chemical_content.replace('list','')
+    chemical_content = chemical_content.replace(')','}')
+    chemical_content = chemical_content.replace('(','{')
+    chemical_content = chemical_content.replace('=','\':')
+
+    chemical_content = chemical_content.replace('/datum/reagent/medicine/','\'')
+    chemical_content = chemical_content.replace('/datum/reagent/drug/','\'')
+    chemical_content = chemical_content.replace('/datum/reagent/consumable/nutriment/','\'')
+    chemical_content = chemical_content.replace('/datum/reagent/consumable/','\'')
+    chemical_content = chemical_content.replace('/datum/reagent/','\'')
+
+
+    chemical_content = chemical_content.replace(':@','=')
+    chemical_content = chemical_content.replace('@','=')
+    chemical_content = chemical_content.replace('={','= {')
+    chemical_content = chemical_content.replace(' : ','\':')
+    return chemical_content.replace(' \'','\'')
+
+
+def add_trait_to_full(full, trait, trait_name, default, inheritance,
+        inherent, tl_naem, inheritance_default):
+
+    if trait_name in inheritance_default and not trait:
+        trait = inheritance_default[trait_name]
+
+    if default and not trait:
+        trait = f'{trait_name} = {default}'
+
+    if trait:
+        if inheritance:
+            inherent[tl_naem][trait_name] = trait
+
+        trait = F"\t{trait}\n"
+        full = full + trait
+
+    return full,trait
+
+
+def burer_piopes(burer, text_buffer, inherent):
```

Comment:
Codacy found an issue: [burer_piopes is too complex (25) (MC0001)](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6388395)